src/test/java/org/prebid/server/bidder/gothamads/GothamAdsBidderTest.java

```diff
@@ -0,0 +1,246 @@
+
+        // then
+        assertThat(result.getErrors())
+                .containsExactly(badServerResponse("Unable to fetch mediaType 3 in multi-format: 3"));
+    }
+
+    private static Stream<Arguments> responsesWithoutSeatBids() {
+        return Stream.of(
+                Arguments.of(givenEmptyBidResponse()),
+                Arguments.of(givenBidResponse())
+        );
+    }
+
+    private static Result<List<HttpRequest<BidRequest>>> expectedResults(BidRequest bidRequest, String... impIds) {
+        return Result.withValue(HttpRequest.<BidRequest>builder()
+                .method(HttpMethod.POST)
+                .uri("https://test-url.com/?pass=accountId")
+                .headers(expectedHeaders())
+                .impIds(impIds.length == 0 ? Set.of(IMP_ID) : Set.of(impIds))
+                .body(jacksonMapper.encodeToBytes(bidRequest))
+                .payload(bidRequest)
+                .build()
+        );
+    }
+
+    private static MultiMap expectedHeaders() {
+        return headers()
+                .set(CONTENT_TYPE_HEADER, APPLICATION_JSON_CONTENT_TYPE)
+                .set(ACCEPT_HEADER, APPLICATION_JSON_VALUE)
+                .set(X_OPENRTB_VERSION_HEADER, OPEN_RTB_VERSION)
+                .set(USER_AGENT_HEADER, DEVICE_UA)
+                .set(X_FORWARDED_FOR_HEADER, List.of(IPV_6_HEADER_VALUE, IP_HEADER_VALUE));
+    }
+
+    private static BidRequest givenBidRequest(Function<Imp.ImpBuilder, Imp.ImpBuilder>... impCustomizers) {
+        return givenBidRequest(identity(), List.of(impCustomizers));
+    }
+
+    private static BidRequest givenBidRequest(
+            Function<BidRequest.BidRequestBuilder, BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            List<Function<Imp.ImpBuilder, Imp.ImpBuilder>> impCustomizers) {
+        return bidRequestCustomizer.apply(
+                        BidRequest.builder()
+                                .device(Device.builder()
+                                        .ua(DEVICE_UA)
+                                        .ip(IP_HEADER_VALUE)
+                                        .ipv6(IPV_6_HEADER_VALUE)
+                                        .build())
+                                .imp(impCustomizers.stream()
+                                        .map(GothamAdsBidderTest::givenImp)
+                                        .toList())
+                )
+                .build();
+    }
+
+    private static Imp givenImp(Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
+        return impCustomizer.apply(Imp.builder().id(IMP_ID).ext(givenExt())).build();
+    }
+
+    private static ObjectNode givenExt() {
+        return mapper.valueToTree(ExtPrebid.of(null, GothamAdsImpExt.of("accountId")));
+    }
+
+    @SneakyThrows
+    private static String givenEmptyBidResponse() {
+        return mapper.writeValueAsString(BidResponse.builder().build());
+    }
+
+    @SneakyThrows
```

Comment:
Please, dont use this annotation. We tend to avoid it. Just replace with `throws` syntax.